XU-TechnicalPaper.md

```diff
@@ -247,44 +251,46 @@ Minimum local local liquidity? Give up on this rule?
+- **We support Lightning/Raiden adoption and growth** with above incentives also help to [solve the maintenance problem of such nodes](https://www.youtube.com/watch?v=dlJG4OHdJzs) (the exchange wants maintain the XU software anyways to be able to use it) 
 
 **Incentives for traders**
 - A user has X trading volume on Exchange Union, this user gets rewarded with XUC from his exchange platform
+- This is more a program of each individual exchange
 
-Each XU node by default opens a channel for XUC in order to be able to pay for channel routing and orderbook relays and also vice-versa receive XUC payments for these services. Also, we could use the XUC payment channel as some sort of emergency fallback if other channel’s volume is not sufficient.
-```
-[TODO]
-Research https://twitter.com/VitalikButerin/status/940429788673536000
-```
+*By default, each XU node opens a XUC payment channel in order to be able to pay for orderbook relays and other services and also vice-versa receive XUC payments for these services.*
 
+`
+[TODO]
+Also, we could use the XUC payment channel as some sort of emergency fallback if other channel’s volume is not sufficient.
+`
 
 3.10. Business Model: XUC
 -------------------------
-Exchange Union’s business model is based on XUC being valuable, which not only makes the incentivisation work, but benefits all stakeholders. XUC’s overall supply is fixed to 3 billion, which naturally makes it deflationary the more the XU ecosystem (& thus demand) grows. This makes XUC a meaningful incentive. The original idea was, that every transaction burns a small amount of XUC as fee, which is needed to prevent spam on a sidechain. However, payment channels don’t face the problem of spamming and thus don’t need a blockchain-like fee. Also, with an artificial burn-per-trade we could get rid of the 30% excess supply (Last ‘reserve’ part of the token allocation in the white paper) in a more elegant way than just burning it at once. A ‘transparency’ section on the website informs about every new XUC release and purpose. Smart contracts will handle the burning.
+Exchange Union’s business model is based on XUC being valuable, which not only makes the incentivization work, but benefits all stakeholders. XUC’s overall supply is fixed to 3 billion, which naturally makes it deflationary the more the XU ecosystem (& thus demand) grows. The original idea was, that every transaction burns a small amount of XUC as fee, which is needed to prevent spam on a sidechain, which was the originally proposed technological solution. However, payment channels don’t face the problem of spamming and thus don’t need a blockchain-like fee for transactions.
```

Comment:
I think this can just read "not only makes the incentives work." The third sentence uses "original" twice, it could read "The original proposal for Exchange Union used sidechains where every transaction would burn a small amount of XUC as a way to combat spam. However, spam transactions are less of a concern given the current solution using payment channels." I think though that there is some concern over spam even in a network like lightning - it's possible to DOS a node by overwhelming it with transactions. The lightning payment fee is supposed to discourage that.

Reply:
Yes. Moreover, Lightening suffers from same flaw that allows mempool spamming in the bitcoin network, where they create an artificial demand for space in a network's mempool. 

> "The original proposal for Exchange Union used sidechains where every transaction would burn a small amount of XUC as a way to combat spam. ***However, the concern about spam transactions are mitigated to good extent by using payment channels"  ***

Reply:
Yes. Moreover, Lightening suffers from same flaw that allows mempool spamming in the bitcoin network, where they create an artificial demand for space in a network's mempool.

 "The original proposal for Exchange Union used sidechains where every transaction would burn a small amount of XUC as a way to combat spam. ***However, the concern about spam transactions are mitigated to good extent by using payment channels" ***